examples/doc/samples/scripts/test_scripts.py

```diff
@@ -34,7 +45,9 @@ def tearDown(self):
     def run_script(self, test, filter, yaml=False):
         cwd = self.cwd+os.sep+test+os.sep
         os.chdir(cwd)
-        pyutilib.subprocess.run(sys.executable+' script.py', outfile=cwd+os.sep+'script.log', cwd=cwd)
+        with open(cwd+os.sep+'script.log', 'w') as f:
+            subprocess.run([sys.executable, 'script.py'], 
+                           stdout=f, stderr=f, cwd=cwd)
```

Comment:
`f` is being opened in text mode; do we need `universal_newlines=True` here?

Reply:
I tried running a `subprocess` command locally using this same format, both with and without `universal_newlines`, and it didn't seem to make any difference. So... I don't think so.